### DISCUSSION TITLE
Mac OS C++14 compiler

### DISCUSSION THREAD

**Discussion by nikero41** [nvim-neorg/neorg#888] [Q&A] [answered]
I get errors when i try to run `sync-parsers`.
I saw in the youtube video that i should install an other compiler.

Any suggestions or guides for which compiler i should install?
(P.S. Not a C++ dev so google search was not really helpful)

**Comment by laher** [comment, marked as answer]
This is what I did:

```bash
brew install gcc
```

I tried setting the `CC` env variable, but it didn't help on its own. So then I added this to my treesitter config, and then it worked. 

```lua
require("nvim-treesitter.install").compilers = { "gcc-13" }
```

It did not work with just `{ "gcc" }`. This command `ls /opt/homebrew/bin/gcc-1*` told me that I have gcc-13. 

I don't know why the extra step was needed. I am also not a C/C++ dev.

**Comment by nikero41** [comment]
Thank you that worked for me as well.
